orangecontrib/text/preprocess.py

```diff
@@ -57,8 +57,11 @@ def __init__(self, lowercase=True, stop_words=None,
                 of expected values/types.
         """
         # Tokenizer.
-        self.tokenizer = word_tokenize
-        if use_twitter_tokenizer:
+        if tokenizer == 'default':
+            self.tokenizer = word_tokenize
+        elif tokenizer == 'no_punct':
+            self.tokenizer = RegexpTokenizer(r'\w+').tokenize
+        elif tokenizer == 'twitter':
             self.tokenizer = None   # TODO: Change when twitter is available.
```

Comment:
Could you also add Twitter tokenizer? It is available in NLTK as [nltk.tokenize.casual.TweetTokenizer](http://www.nltk.org/api/nltk.tokenize.html#module-nltk.tokenize.casual). Not sure how we missed this before 🙈